omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -41,155 +51,196 @@ running::
     $ javac -version
     javac 1.8.0_51
 
-Requirements
+
+OSX Basics
 ------------
 
-.. _`Homebrew wiki`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md
+In order to develop on OMERO, we recommend you ensure you have your MAC setup for
+developement. The first step to achieving this is to create a .bash_profile file in the
+root directory of your user folder.
 
-All the requirements for OMERO will be installed under :file:`/usr/local`. See also: Installation instructions on the `Homebrew wiki`_.
+To create a .bash_profile from terminal, if one does not already exist::
 
-Install Homebrew and make sure :file:`/usr/local/bin` is prepended to your
-:envvar:`PATH`::
+    $ touch ~/.bash_profile
 
-    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-    $ export PATH=/usr/local/bin:$PATH
+To open your .bash_profile in a text editor, such as the built in TextEdit app, use::
 
-Update Homebrew and run 'brew doctor' to fix potential issues beforehand::
+    $ open -a TextEdit.app ~/.bash_profile
 
-    $ brew update
-    $ brew doctor
+.. note::
+   If you want to see changes to your .bash_profile take affect without restarting
+   OSX, run::
 
-Install git if not already present::
+   $ source ~/.bash_profile
 
-    $ brew list | grep "\bgit\b" || brew install git
+Requirements
+------------
 
-Install PostgreSQL database server::
+1. Open a command-line terminal and install git if not already present::
+
+    $ brew install git
+
+2. Install PostgreSQL database server::
 
-    $ export LANG=${LANG:-en_US.UTF-8}
-    $ export LANGUAGE=${LANGUAGE:-en_US:en}
     $ brew install postgresql
 
-.. _`Homebrew and Python`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md
+   To ensure PostgreSQL uses UTF-8 encoding, open your bash profile and 
+   add the following enviorment variables::
+
+    export LANG=en_US.UTF-8
+    export LANGUAGE=en_US:en
 
-You should install OMERO using Python 2.7 provided by
-Homebrew since it makes using Homebrew-provided modules
-simpler, for example the Ice python bindings needed by OMERO. For a
-more thorough description of the Homebrew solution, see the `Homebrew
-and Python`_ page. Note that the automated script linked above tests
-the OMERO installation using the Homebrew Python.
+3. OMERO depends on ICE 3.6 and unfortuently does not run with 
+   the latest version of ICE at this time (ICE 3.7.3). To obtain 
```

Comment:
:+1: but do we need a reminder somewhere? Will we grep for `3.6` and `3.7` globally for OMERO 5.5?